lib/loaders/sass.js

```diff
@@ -15,14 +15,14 @@ const applyOptionsCallback = require('../utils/apply-options-callback');
 
 /**
  * @param {WebpackConfig} webpackConfig
- * @param {Object} sassOption Options to pass to the loader
+ * @param {bool} useCssModules
  * @return {Array} of loaders to use for Sass files
  */
 module.exports = {
-    getLoaders(webpackConfig, sassOptions = {}) {
```

Comment:
No point keeping a `sassOptions` parameter here, the only allowed option is `resolveUrlLoader` which is already handled [here](https://github.com/symfony/webpack-encore/pull/511/files#diff-546e3b877b44d48c5df096497b898ce7L40) by directly looking into the `webpackConfig` object.